charts/prometheus-node-exporter/Chart.yaml

```
@@ -6,8 +6,8 @@ keywords:
   - prometheus
   - exporter
 type: application
-version: 4.22.0
-appVersion: 1.6.0
+version: 4.23.0
```

I'd suggest to raise the patch number in this case.

```suggestion
version: 4.22.1
```